screenpy_requests/questions/body_of_the_last_response.py

```diff
@@ -15,20 +17,49 @@
 class BodyOfTheLastResponse:
     """Ask about the body of the last API response received by the Actor.
 
+    If you are expecting a JSON response, this Question can be indexed to
+    return only a section of that response. See the examples below.
+
+    If you are expecting a text response, you can also slice the text.
+
     Abilities Required:
         :class:`~screenpy_requests.abilities.MakeAPIRequests`
 
     Examples::
 
+        # JSON response
         the_actor.should(
             See.the(BodyOfTheLastResponse(), ContainsTheEntry(play="Hamlet"))
         )
 
+        # JSON response, picked
+        the_actor.should(
+            See.the(
+                BodyOfTheLastResponse()["users"][0]["first_name"],
+                ReadsExactly("Monty")
+            ),
+        )
+
+        # text response
         the_actor.should(
             See.the(BodyOfTheLastResponse(), ReadsExactly("To be, or not to be"))
         )
+
+        # text response, sliced
+        the_actor.should(
+            See.the(BodyOfTheLastResponse()[6:12], ReadsExactly("Python"))
+        )
     """
 
+    body_parts: list[str | int]
+
+    def __getitem__(self, key: str | int) -> BodyOfTheLastResponse:
```

Comment:
Add `slice` to `key` and `body_parts` annotations
Mypy will bitch about the following:
`foo = BodyOfTheLastResponse()["slideshow"]["slides"][1]["items"][0:2]`